src/domain/cold-storage/index.ts

```diff
@@ -8,7 +8,7 @@ export const RebalanceChecker = ({
   const getWithdrawFromHotWalletAmount = ({
     onChainHotWalletBalance,
     offChainHotWalletBalance,
-  }) => {
+  }: RebalanceCheckerArgs): Satoshis => {
```

Comment:
Shouldn't this be `WithdrawFromHotWalletAmountArgs`? Is the type being used somewhere else?

Reply:
good question. will have a look and eventually incorporate the change in my next PR